src/App.js

```diff
@@ -0,0 +1,33 @@
+import './App.css';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/head';
+import Rockets from './Routes/rockets';
+import Missions from './Routes/missions';
+import Myprofile from './Routes/myprofile';
+
+import { fetchMissions } from './redux/mission/missionSlice';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
```

Comment:
You have done great dispatching the fetchMissions action. However, it's required that the action for fetching missions be dispatched only when the user navigates to the missions page (and only once). Kindly dispatch the action within the missions component in order to achieve this.